_posts/con-commands/2019-08-23-mom_restart.md

```diff
@@ -0,0 +1,31 @@
+---
+title: mom_restart
+category: command
+tags:
+  - timer
+  - teleport
+optional_params:
+  - Track number
+---
+
+Restarts the player to the start trigger. Optionally takes a track number (default is main track).
+
+## Usage Examples
+
+> `mom_restart -1`
+
+This will teleport the player to the spawn point.
+
+> `mom_restart 0` or `mom_restart`
+
+This will teleport the player to the start of the main track (track 0).
```

Comment:
If no track is specified, [the current track is used](https://github.com/momentum-mod/game/blob/develop/mp/src/game/server/momentum/mom_timer.cpp#L402).

Reply:
Woops! I need to play the game more :p 
Fixed that up now :)